--- EPICS-CPP-4.6.0/pvaPy/src/pvaccess/PvObject.cpp	2016-09-18 10:22:36.000000000 -0700
+++ EPICS-CPP-4.6.0/pvaPy/src/pvaccess/PvObject.cpp	2017-03-03 15:12:12.804982800 -0800
@@ -16,7 +16,7 @@
 #include "boost/python/stl_iterator.hpp"
 
 #if defined HAVE_BOOST_NUM_PY && HAVE_BOOST_NUM_PY == 1
-#include "boost/numpy.hpp"
+#include "boost/python/numpy.hpp"
 const bool PvObject::UseNumPyArraysDefault(true);
 #else
 const bool PvObject::UseNumPyArraysDefault(false);
@@ -751,7 +751,7 @@
 bool PvObject::boostNumPyInitialized(initializeBoostNumPy());
 bool PvObject::initializeBoostNumPy() 
 {
-    boost::numpy::initialize();
+    boost::python::numpy::initialize();
     return true;
 }
 
