--- pvaClientCPP/configure/CONFIG_SITE  2016-09-18 10:22:35.000000000 -0700
+++ pvaClientCPP/configure/CONFIG_SITE  2017-03-09 15:58:53.277698400 -0800
@@ -18,7 +18,7 @@

 # To install files into a location other than $(TOP) define
 #   INSTALL_LOCATION here.
-#INSTALL_LOCATION=</path/name/to/install/top>
+INSTALL_LOCATION=$(PREFIX)/epics-v4/pvaClientCPP

 -include $(TOP)/../CONFIG_SITE.local
 -include $(TOP)/configure/CONFIG_SITE.local
