--- src/H5Pfapl.c.orig	2015-05-28 09:01:47.000000000 -0700
+++ src/H5Pfapl.c	2015-06-05 17:16:43.075228397 -0700
@@ -64,7 +64,7 @@
 #define H5F_ACS_DATA_CACHE_NUM_SLOTS_DEF        521
 /* Definition for size of raw data chunk cache(bytes) */
 #define H5F_ACS_DATA_CACHE_BYTE_SIZE_SIZE       sizeof(size_t)
-#define H5F_ACS_DATA_CACHE_BYTE_SIZE_DEF        (1024*1024)
+#define H5F_ACS_DATA_CACHE_BYTE_SIZE_DEF        (101*1024*1024)
 /* Definition for preemption read chunks first */
 #define H5F_ACS_PREEMPT_READ_CHUNKS_SIZE        sizeof(double)
 #define H5F_ACS_PREEMPT_READ_CHUNKS_DEF         0.75f
