--- h5py/h5p.pyx.old	2015-01-14 13:41:09.000000000 -0800
+++ h5py/h5p.pyx	2016-03-24 15:19:53.804500853 -0700
@@ -25,7 +25,7 @@
 from ._objects import phil, with_phil
 
 if MPI:
-    from mpi4py.mpi_c cimport MPI_Comm, MPI_Info, MPI_Comm_dup, MPI_Info_dup, \
+    from mpi4py.libmpi cimport MPI_Comm, MPI_Info, MPI_Comm_dup, MPI_Info_dup, \
                                MPI_Comm_free, MPI_Info_free
 
 # Initialization
