--- src/H5Zshuffle.c.orig	2015-05-28 09:01:48.000000000 -0700
+++ src/H5Zshuffle.c	2015-06-05 17:20:39.964073357 -0700
@@ -86,6 +86,11 @@ H5Z_set_local_shuffle(hid_t dcpl_id, hid
     if(H5P_get_filter_by_id(dcpl_plist, H5Z_FILTER_SHUFFLE, &flags, &cd_nelmts, cd_values, (size_t)0, NULL, NULL) < 0)
 	HGOTO_ERROR(H5E_PLINE, H5E_CANTGET, FAIL, "can't get shuffle parameters")
 
+    /* If object is an array use its base type */
+    while (H5T_get_class(type, FALSE) == H5T_ARRAY) {
+	type = H5T_get_super(type);
+    }
+
     /* Set "local" parameter for this dataset */
     if((cd_values[H5Z_SHUFFLE_PARM_SIZE] = (unsigned)H5T_get_size(type)) == 0)
 	HGOTO_ERROR(H5E_PLINE, H5E_BADTYPE, FAIL, "bad datatype size")
